docs/pages/blog/2020.md

```diff
@@ -0,0 +1,115 @@
+
+### Branding
+
+Up until now, we never had a mindful reflection on what the branding of Material-UI should be. We picked a color from the 2014 Material Design palette, and that's it.
+The material-ui.com marketing website will soon wear a brand new style! We need to create our own identity, distinct from Material Design.
+
+### Material-UI X
+
+We have started to deliver advanced React components in 2020 with the data grid, including a [commercial version](/components/data-grid/#commercial-version) and the beginning of a [date range picker](https://next.material-ui.com/components/date-range-picker/).
+We will double down on these existing components as long as necessary to have them find the market.
+
+By the end of 2021, we aim to have released these components as stable, implement all the [features planned](/components/data-grid/getting-started/#feature-comparison), and at least double the size of the team.
+
+### Material-UI v5
+
+We will release the next major iteration of the library. A highlight of the key improvements coming ✨:
+
+- Polish and promote most of the components that were in the lab in v4 to the core.
+- A new styled engine. Migrate from JSS to emotion (default) and styled-components's `styled` API.
+- Improve customizability. Add new powers to the theme with dynamic color & [variant](https://next.material-ui.com/customization/typography/#adding-amp-disabling-variants) support. Add a new [`sx` prop](https://next.material-ui.com/system/basics/) for quick customizations to all the components. Expose global class names. Deprecate the `makeStyles` and `withStyles` API.
+- Breaking changes on the API to make it more intuitive.
+- Add full support for React strict mode. We recommend to enable it.
+- Improve the performance of the system by a x3-x5 [factor](https://github.com/mui-org/material-ui/issues/21657#issuecomment-707140999).
+- Reduce bundle size: split IE11 into a different bundle [-6kB](https://github.com/mui-org/material-ui/pull/22814#issuecomment-700995216), migrate to emotion [-5kB](https://github.com/mui-org/material-ui/pull/23308#issuecomment-718748835), Popper.js v2 upgrade [-700B](https://github.com/mui-org/material-ui/pull/21761#issuecomment-657135498).
+- Improve the documentation website: search shortcut, page rating, fast material icons copy button, etc.
+
+And [much more](https://github.com/mui-org/material-ui/issues/20012).
+
+### A second theme
+
+While Material Design is a predominant design choice for building rich client-side applications, it's not in the landscape of landing and marketing pages.
+This use case for React has yet to win a large-scale adoption by the community but the traction of Next.js is a strong signal that it has the potential to happen. React pushing [server-side components](https://reactjs.org/blog/2020/12/21/data-fetching-with-react-server-components.html), with zero bundle size overhead could accelerate it.
```

Comment:
What's the use-case for server side components on a home page, landing page, or marketing pages, that will cause an upswing in the adoption of React for this use-case?

Isn't it fair to say that HTML and CSS are a far better fit, and always will be?

Reply:
> Isn't it fair to say that HTML and CSS are a far better fit, and always will be?

I think that there is little place for JavaScript in there. Its usage should be limited, with a couple of carousel, tooltip, scrollspy, menu, popup, slider, or combo box. The new server-side components deliver zero JavaScript bundle size, it's effectively HTML and CSS. It can replace slim (rails), twig (Symfony), blade (laravel), etc. However, unlike jQuery style, you get components with isolated client-side logic for the few places where you need it.

IMHO, Next.js wouldn't be growing if HTML **and** CSS raw was great (Gatsby is losing over Next.js)

Reply:
https://twitter.com/rauchg/status/1342963266004258817/photo/1 😆

Reply:
Okay, sound like we agree. Optimising the second theme for home page usage doesn't make much sense.

Reply:
> Okay, sound like we agree. Optimising the second theme for home page usage doesn't make much sense.

In our survey, 6 months ago, we go about 20% of the usage of React [for landing pages](https://material-ui.com/blog/2020-developer-survey-results/#16-what-delivery-mechanism-are-you-using). I agree that it shouldn't be the primary factor to optimize with.

However, considering the 20%, that it's growing, that server-side components, might make it explodes in the next 5 years and that Material Design is weak in this segment, I think that we should give it a low weight but still make sure that it works.

Reply:
I don't know how you anticipate optimising the design of the majority of components for use on the homepage when only a minority, many of which have no visible design aspect, have any use there.